test/recipes/30-test_evp_data/evpmd_blake.txt

```diff
@@ -54,6 +54,16 @@ Digest = BLAKE2s256
 Input = 000102030405060708090A0B0C0D0E0F101112131415161718191A1B1C1D1E1F202122232425262728292A2B2C2D2E2F303132333435363738393A3B3C3D3E3F404142434445464748494A4B4C4D4E4F505152535455565758595A5B5C5D5E5F606162636465666768696A6B6C6D6E6F707172737475767778797A7B7C7D7E7F8081
 Output = C80ABEEBB669AD5DEEB5F5EC8EA6B7A05DDF7D31EC4C0A2EE20B0B98CAEC6746
 
+Digest = BLAKE2s256
+Input =
+OutputSize = 16
+Output = 64550d6ffe2c0a01a14aba1eade0200c
+
+Digest = BLAKE2s256
+Input = 61
+OutputSize = 10
+Output = b60d322755eebca92b5e
+
```

Comment:
If there is an upper bound on the output length there should be failure checks here..

Reply:
Similarly, this just mimicks the blake2b tests at the end of the file (but obviously with blake2s outputs). If the limit in the formerly-blake2b code is appropriate, then maybe. No limit is documented afaict